cni/pkg/ambient/net_linux.go

```diff
@@ -245,12 +245,11 @@ func getDeviceWithDestinationOf(ip string) (string, error) {
 	return link.Attrs().Name, nil
 }
 
-func GetIndexAndPeerMac(podIfName, ns string) (int, net.HardwareAddr, error) {
+func GetIndexAndPeerMac(podIfName, nspath string) (int, net.HardwareAddr, error) {
```

Comment:
do we have a test for it, why not the caller of changed

Reply:
Yes, we've tested it with both kind and cri-dockerd env. This seems the CNI/CRI conventions. for instance, calico considers the `Netns` as nspath as well.  https://github.com/projectcalico/calico/blob/8d31b2669e8cde1558b128fc7db3015970921e90/cni-plugin/pkg/dataplane/linux/dataplane_linux.go#L85